### DISCUSSION TITLE
[Post] Diferencia entre Fork & Clone

### DISCUSSION THREAD

**Discussion by DanielOe21** [LaunchX-InnovaccionVirtual/CursoIntroPython#198] [Show and tell]
# Fork vs Clone

Esto lo puse a manera de respuesta en una pregunta aquí mismo, pero me pareció buena idea compartirlo aparte para que los demás pudieran verlo de primera mano.

Básicamente, ambas son formas de "copiar" un repositorio, pero una manera sencilla para entenderlo sería:

- Fork es una copia que sigue estando ´ligada´ al repositorio original. Si la original sufre cambios, la copia la puedes actualizar (manualmente). Puedes hacer cambios en tu Fork y hacer una solicitud para integrar los cambios al original.

- Clone es una copia exacta del repositorio en ese preciso momento en el que lo hagas, pero ya no está ligada al original, puedes hacer con él lo que quieras sin afectar al otro.

Follow me!
<a href="https://github.com/DanielOe-21">
<img alt="GitHub followers" src="https://img.shields.io/github/followers/DanielOe-21?style=social">
</a>
<img align='center' src="https://media.giphy.com/media/NytMLKyiaIh6VH9SPm/giphy.gif" width="240">

**Comment by mikaelzen** [comment]
Muy útil la información, gracias! Y ya te sigo 🚀
